services/apps/alcs/src/providers/typeorm/migrations/1707851589615-add_advanced_parcel_mapping.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAdvancedParcelMapping1707851589615
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    UPDATE
+        alcs.application_submission
+    SET
+        has_other_parcels_in_community = 'False';
+    WITH parcel_count as (
+        SELECT count(*), ap.application_submission_uuid
```

Comment:
To make a decision that the other parcel exists for submission you will need to check OATS DB since ETL did not bring over other parcels to ALCS

Reply:
placing this on hold until mapping is created. ticket unclear